src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue

```diff
@@ -0,0 +1,233 @@
+<template />
+
+<script setup lang="ts">
+import { onMounted, watch } from "vue";
+import { declareModule } from "@/composable/useModule";
+import { declareConfig } from "@/composable/useSettings";
+import { hiddenElementSettings } from "./hiddenElements";
+
+const { markAsReady } = declareModule("hidden-elements", {
+	name: "Hidden Elements",
+	depends_on: [],
+});
+
+function updateHiddenElements(): void {
+	hiddenElementSettings.forEach((setting) => {
+		document.body.classList.toggle(setting.class, setting.isHidden.value);
+	});
+}
+
+onMounted(() => {
+	updateHiddenElements();
+
+	watch(
+		() => hiddenElementSettings,
+		() => {
+			updateHiddenElements();
+		},
```

Comment:
This would be better using `watchEffect()` instead of `onMounted()`. The inner watch and outer function is unnecessary

Reply:
Thanks for this suggestion, still learning about Vue reactivity so I thought there would be a better way. watchEffect works perfectly.